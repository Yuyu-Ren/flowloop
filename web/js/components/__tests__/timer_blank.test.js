/* eslint-env jest */

import TimerBlank from '../timer_blank'

import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

describe('TimerBlank', () => {
  it('exists', () => {
    expect(TimerBlank).toBeTruthy()
  })

  it('renders', () => {
    const comp = <TimerBlank />
    const wrapper = shallow(comp)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
