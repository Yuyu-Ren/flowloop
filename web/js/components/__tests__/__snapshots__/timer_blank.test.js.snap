// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimerBlank renders 1`] = `
<TimerLayout>
  <SideEffect(DocumentTitle)
    title="No timer"
  />
  <div
    className="timer-heading"
  >
    No timer active.
  </div>
  <div
    className="timer-spacer"
  />
  <div
    className="timer-actions"
  >
    <button
      className="timer-button button -back"
      onClick={[Function]}
    >
      Back
    </button>
  </div>
</TimerLayout>
`;
